mcs/build/library.make

```diff
@@ -258,7 +258,7 @@ endif
 # make dist will collect files in .sources files from all profiles
 dist-local: dist-default
 	subs=' ' ; \
-	for f in `$(topdir)/tools/removecomments.sh $(filter-out $(wildcard *_test.dll.sources) $(wildcard *_xtest.dll.sources) $(wildcard *exclude.sources),$(wildcard *.sources))` $(TEST_FILES) ; do \
+	for f in `$(topdir)/tools/removecomments.sh $(filter-out $(wildcard *_test.dll.sources) $(wildcard *_xtest.dll.sources) $(wildcard *exclude.sources),$(wildcard *.sources))` ; do \
```

Comment:
Is this fixing a bug?

Reply:
it's not needed since those files are already captured by  the`$(wildcard *_test.dll.sources)` and I wanted to clean it up while I still understand how it works :)

Reply:
Hmm turns out it _is_ needed because the nunit test .sources use the magic behavior where Test/ is prepended to the path so I put it back. It's still slightly broken though since `$(TEST_FILES)` doesn't resolve any includes but that was already the case.